packages/ui-components/src/components/range/range.tsx

```diff
@@ -48,7 +48,7 @@ export class KvRange implements IRange, IRangeEvents {
 		return (
 			<Host>
 				<div class="range-container">
-					<input id="slider" class="slider" type="range" min={this.min} max={this.max} value={this.value} step={this.step} onInput={this.applyCssStyles} />
+					<input id="slider" class="slider" type="range" min={this.min} max={this.max} step={this.step} value={this.value} onInput={this.applyCssStyles} />
```

Comment:
why was the order of the args changed? was it to keep min, max and step together?

Reply:
I thought it was because eslint was complaining. However, I just tested it and it is not anymore. I guess I will revert it and let PR #137 deal with warnings not related to my changes :+1:

Reply:
I mean, from my point of view the change you made makes it a bit more organized, let's see what everyone else thinks :)

Reply:
LGTM this way 👍